constantine/serialization/codecs_banderwagon.nim

```diff
@@ -123,7 +123,7 @@ func deserialize_unchecked*(dst: var EC_Prj, src: array[32, byte]): CttCodecEccS
   var x{.noInit.}: Fp[Banderwagon]
   x.fromBig(t)
 
-  let onCurve = dst.trySetFromCoordX(x)
+  let onCurve = dst.trySetFromCoordX_vartime(x) # later to be shifted to a constant time version
```

Comment:
```suggestion
  let onCurve = dst.trySetFromCoordX_vartime(x) # later to be shifted to a constant time version
```

deserialized_unchecked should be duplicated into:
- deserialized_unchecked_vartime, and the "# later to be shifted to a constant time version" is unnecessary
- deserialized_unchecked, that uses the regular `trySetFromCoordX`

Then if the constant-time is fast enough, we can deprecate the vartime version to keep the API simple,
if it's too slow, we can maintain both.